Storage/Storage/Model/ShippingLabel+CoreDataProperties.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+import CoreData
+
+
+extension ShippingLabel {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<ShippingLabel> {
+        return NSFetchRequest<ShippingLabel>(entityName: "ShippingLabel")
+    }
+
+    @NSManaged public var siteID: Int64
+    @NSManaged public var orderID: Int64
+    @NSManaged public var shippingLabelID: Int64
+    @NSManaged public var carrierID: String
+    @NSManaged public var dateCreated: Date
+    @NSManaged public var packageName: String
+    @NSManaged public var rate: Double
+    @NSManaged public var currency: String
+    @NSManaged public var trackingNumber: String
+    @NSManaged public var serviceName: String
+    @NSManaged public var refundableAmount: Double
+    @NSManaged public var status: String
+    @NSManaged public var productIDs: [Int64]?
```

Comment:
I wish this was not optional. I somehow find it unnecessarily tricky to deal with optional arrays. But no worries.

Reply:
Hmm actually, I'll make this non-optional. I agree that optional arrays are a bit unnecessary since we can often check whether it's empty (unless an empty array has other meaning). The readonly model in the networking layer actually has this property non-optional as well 👍

Reply:
Updated here: https://github.com/woocommerce/woocommerce-ios/pull/3168/commits/9784a6e40078766f2801cbc3754ca1ecbdc7e0fc